index.js

```diff
@@ -23,7 +23,7 @@ for (const f of project.getSourceFiles("**/*.d.ts")) {
     for (const s of ss) {
         const g = s.getGetAccessor();
         if (!g) {
-            s.replaceWithText(`${s.getName()}: ${s.getType().getText()}`)
+            s.replaceWithText(`${s.getName()}: ${s.getReturnTypeNodeOrThrow().getText()}`)
```

Comment:
This commit is wrong. See https://github.com/sandersn/downlevel-dts/pull/4/commits/6dd7ba4da4a7a04800e502c8401656f745dbaa08 for fix.